src/runtime_src/core/common/api/xrt_kernel.cpp

```diff
@@ -1316,6 +1316,7 @@ class kernel_impl : public std::enable_shared_from_this<kernel_impl>
   size_t num_cumasks = 1;              // Required number of command cu masks
   control_type protocol = control_type::none; // Default opcode
   uint32_t uid;                        // Internal unique id for debug
+  uint32_t m_ctrl_code_index = 0;      // Index to identify which ctrl code to load in elf
```

Comment:
Thanks for using `m_`, I have long regretted not having started out that way for all the other members.

Reply:
Thanks Soren